datafusion/sqllogictest/test_files/spark/datetime/make_dt_interval.slt

```diff
@@ -64,10 +64,10 @@ SELECT (make_dt_interval(0, 0, 0, null))
 NULL
 
 # missing params
-query ?
+query error does not support zero arguments
 SELECT (make_dt_interval()) AS make_dt_interval
-----
-0 days 0 hours 0 mins 0.000000 secs
+
```

Comment:
This actually does support zero arguments so we need to maintain that previous behaviour